docs/website/docs/pipelines/stripe.md

```diff
@@ -42,20 +42,21 @@ To get started with your data pipeline, follow these steps:
 dlt init stripe_analytics bigquery
 ```
 
-This command will initialize your pipeline with Stripe as the source and BigQuery as the destination. If you'd like to use a different destination, simply replace **`bigquery`** with the name of your preferred destination. You can find supported destinations and their configuration options in our [documentation](https://dlthub.com/docs/destinations/duckdb)
+This command will initialize your verified source with Stripe as the source and BigQuery as the destination. If you'd like to use a different destination, simply replace **`bigquery`** with the name of your preferred destination. You can find supported destinations and their configuration options in our [documentation](https://dlthub.com/docs/destinations/duckdb)
```

Comment:
IMO: This command will initialize your verified source with Stripe and creates pipeline example with BigQuery as the destination.